### DISCUSSION TITLE
[Suggestions] Combat screen, Action bar, display at least the first troop that will act the next turn

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#5175] [Ideas]
In fheroes2, I suggest to display at least the first troop that will act the next turn.
Ideally, several of the troops that will act the next turn should be displayed, with at least the first troop of each opponent.

In fheroes2:

![Army order bar 001](https://user-images.githubusercontent.com/43583994/159983006-2e22fedb-4c3c-4e58-b793-314a7b519f14.png)

Related issue: https://github.com/ihhub/fheroes2/issues/5166.

**Comment by ihhub** [comment]
I think we need to fully redo this feature as it looks yuckie🙂
